kong/runloop/upstream_ssl.lua

```diff
@@ -42,7 +42,7 @@ local function set_service_ssl(ctx)
   end
 
   local tls_verify = service.tls_verify
-  if tls_verify then
+  if tls_verify ~= nil then
```

Comment:
here may be hard to understand, whatever service.tls_verify` is `true` or `false`, we need to `set_upstream_ssl_verify`, only if the user didn't config `service.tls_verify`, we don't need to `set_upstream_ssl_verify`.

and the service `tls_verify` means `Whether to enable verification of upstream server TLS certificate`, I think we should update docs and description: https://github.com/Kong/kong/blob/9d8933d849e24d92037c623d07a0793dbaac3c5d/kong/db/schema/entities/services.lua#L45

Reply:
How about adding a description to mention that if `tls_verify` is not set, the system level config `proxy_ssl_verify` will be used?

Reply:
useful